packages/fluentui/perf/gulp/perf.ts

```diff
@@ -1,5 +1,6 @@
 import express from 'express';
```

Comment:
## [🕵 FluentUI-v0 Open the Visual Regressions report to inspect the 1 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26421&commitId=7831b13880c456b2e2132627d89d602ec25e6976&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1259 screenshots unchanged, 0 screenshots with different dimensions and 1 screenshots with visible difference.


  <details><summary><b>unknown</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Checkbox.Checkbox Example Label Flex Column.chromium.png|22| Changed|

</details>


<div id="vrtCommentFluentUI-v0"/>